tests/test_resources/test_data.py

```diff
@@ -37,6 +38,14 @@
     ConsoleProperties("INFO", True, 130),
 )
 
+RUN_PROPERTIES_PROD = dataclasses.replace(
```

Comment:
Ah! I didn't know this feature 👍

Reply:
I was looking for something like scala's `copy()` :)